src/app/core/transaction-mapper/impl/revolut-transaction-mapper.ts

```diff
@@ -16,10 +16,11 @@ type RevolutTransaction = {
 
 export class RevolutTransactionMapper extends TransactionMapper {
   async map(data: RevolutTransaction[]) {
-    const transactions: CreateTransactionDto[] = [];
+    //TODO: Temp fix for now, but let's not use InputTypes as types for variables
```

Comment:
I agree,

but InputTransactionCreate type is used here to be able to call class-validator's validate(), since InputTransactionCreate has all the validation decorators

can maybe move the class-validator decorators to the Transaction ObjectType or Entity?

Reply:
I will move the class validator decorators to the entity.